sklearn/model_selection/_split.py

```diff
@@ -504,6 +504,10 @@ class GroupKFold(_BaseKFold):
     --------
     LeaveOneGroupOut : For splitting the data according to explicit
         domain-specific stratification of the dataset.
+
+    StratifiedKFold : Takes group information into account to avoid building
+        folds with imbalanced class distributions (for binary or multiclass
+        classification tasks).
```

Comment:
Is "proportions" is more adapted here?
```suggestion
    StratifiedKFold : Takes group information into account to avoid building
        folds with imbalanced class proportions (for binary or multiclass
        classification tasks).
```

Reply:
Actually, taking the change you made l.425 into account:
```suggestion
    StratifiedKFold : Takes class information into account to avoid building
        folds with imbalanced class proportions (for binary or multiclass
        classification tasks).
```